packages/botonic-nlp/src/preprocess/preprocessor.ts

```diff
@@ -1,4 +1,8 @@
 import { Locale } from '../types'
+import { getNormalizer } from './engines/normalizer'
+// import { getStemmer } from './engines/stemmer'
```

Comment:
we should think of a way to activating stemmers depending on the use case (dev is using word embeddings or not). It's a discussion we need to have

Reply:
I am agree, we need a way of choosing which engines are enabled. 
Maybe a class attribute for each engine, something like: `isTokenizerEnabled`, `areStopwordsEnabled`, etc.

Reply:
passing an interface with optional arguments is a flexible way to customize this.
btw, @marcrabat @elozano98  I'm not a fan of calling flags starting with is/are. Writing more is always worse than written less.
The convention to start function names with is/are is to make clear that the function is a predicate (it returns boolean). In this case, maybe enableTokenizer or tokenizeInput would be better. As usual, most important is consistent within the project

Reply:
I agree with Dani 👍

Reply:
Okay, I'll do it in another PR. Thank you both!